pieforproviders/pieforproviders
# Fixes the absences logic

Fixes the absences logic so it'll stop deleting absences when they're created, cleans up some payment bleedover from atlassian work.

## 💅🏼 What issue does this fix?



## 🍂 Before You Submit


## 🛷 Deployment

* [X] Rake Task: `bundle exec rails fix_absences`

## 🧳 Steps to test
- Clean and seed your db, add some attendances with the attendance rake task
- Check how many attendances you have and how many service days
- run `bundle exec rails fix_absences` - absences should get populated, 1:1 with ServiceDays, meaning for every absence created, one and only one ServiceDay should be created
